### PR TITLE
lightning(dm): add current speed in status

### PR DESCRIPTION
**Pull request by lance6716**
Signed-off-by: lance6716 <lance6716@gmail.com>

### What problem does this PR solve?

Issue Number: ref #7115

### What is changed and how it works?

the speed has unit of "bytes per seconds", and will display the old value if "finished bytes" is not updated.

#### Tests 

 - Integration test

### Release note 

```release-note
Add current speed to load unit status.
```

### CODE REVIEW

**Review by D3Hunter — APPROVED** [review]

**Review by GMHDBJD — APPROVED** [review]
LGTM

File: proto/canal/CanalProtocol.pb.go [hunk 1]
```diff
@@ -118,20 +118,23 @@ func (PacketType) EnumDescriptor() ([]byte, []int) {
 }
 
 type Packet struct {
-	//[default = 17];
```
[the remark this answers is not in the document]
Comment (lance6716) [reply proto/canal/CanalProtocol.pb.go:121]: I just run make entry. Maybe format tool hook is also invoked.

**Pull request merged**